reark/src/test/java/io/reark/reark/utils/RxViewBinderTest.java

```diff
@@ -63,19 +65,18 @@ public void setUp() {
         MockitoAnnotations.initMocks(this);
         // Mocking AndroidSchedulers.mainThread() as loopers are not mocked by android unit tests
         PowerMockito.stub(PowerMockito.method(AndroidSchedulers.class, "mainThread"))
-                    .toReturn(Schedulers.immediate());
+                    .toReturn(Schedulers.single());
 
-        testSubject = PublishSubject.create();
+        testSubject = PublishProcessor.create();
 
         binder = new RxViewBinder() {
             @Override
-            protected void bindInternal(@NonNull final CompositeSubscription compositeSubscription) {
+            protected void bindInternal(@NonNull final CompositeDisposable compositeDisposable) {
                 assertNull(testSubscriber);
 
                 // It is not possible to subscribe multiple with the same subscriber.
                 testSubscriber = new TestSubscriber<>();
-
-                compositeSubscription.add(testSubject.subscribe(testSubscriber));
+                compositeDisposable.add(testSubject.subscribe(testSubscriber::onNext));
```

Comment:
The reason why tests here are failing is because before all the events (next, error, complete, subscribe, unsubscribe) were passed to the subscriber. Now you are just passing on next, so that is why for example the unbind does not work as the cancel event is not passed to it. You need to pass all the events to the subscriber